novice-py-getting-started.Rmd

```diff
@@ -0,0 +1,335 @@
+For example: 
+
+```{python}
+message = "It is a far, far better thing that I do, than I have ever done"
+name_length = len("Sydney Carton")
+```
+
+These *variables* can then be used later in functions like `print()`: 
+
+```{python}
+print("My message:", message)
+print("Letters in name:", name_length)
+```
+
+On the top right pane, there's a 
+tab that says "Variable Explorer". If you click on that, you'll see the 
+variables you just created, with some additional information about the *Type* 
+of information - `str` (string), `int` (integer), `float` (decimal point 
+number) and more. 
+
+## Getting Help
+
+However, that top-right pane can also give us additional information; one of 
+the other tabs says "Help". The main way to look up a a particular function 
+or type is to type it into the Console and then type the Help shortcut (which 
+is `CTRL-I`). So, if you wanted to look up information about the `len()` 
+function, we used above, you can type the following into the console and 
+then use the help shortcut: 
+
+```{python}
+len
+```
+
+Typing the full name of the function with the Help tab selected will also 
+trigger the help page about that function: 
+
+```{python}
+len()
+```
+
+Sometimes, however, your questions won't fit well within the built-in help pages, 
+available through Spyder. This often happens when you need to do something 
+completely new and you don't know where to start. In that scenario, there are 
+multiple internet resources that are helpful:
+
+* blogs
+* question pages on Stack Overflow
+* Twitter
+
+Sometimes these resources can be overwhelming and confusing! Some things to 
+look for on a Stack Overflow answer: 
+
+* Is the question clearly stated and is there example code? If you're 
+debugging, does the example code look like yours? 
+* Answers have upvotes and downvotes. Is there one clear answer that has a lot 
+of upvotes in response to a question? 
+* How complex is the answer? While some questions will necessarily have a 
+complicated answer, for many common programming tasks, there should be a 
+few-line solution to the question. 
+* Do you recognize any terms in the solution? If you don't, are there other 
+terms that you could search on? 
+
+## Data on a Computer
+
+In the beginning of this chapter, we talked about learning to work with 
+*programs*, and *data*. We've now covered many aspects of our new 
+programming environment, so this chapter will conclude with a short discussion 
+of data. 
+
+Data is simply information that's been stored in some permanent form. One 
+could think of family photos and old credit card statements as data about a 
+family over time. In the context of data analysis via programming, we're 
+mostly going to be concerned with text-based data that has been saved as 
+a file on your computer. 
+
+When you have a data file on your computer, it's actually saved on your 
+computer's hard drive or "disk". This is a componenent of the computer that
```

Comment:
again, ref to glossary, and possibly explain why it's (still) called a disk.

Reply:
wasn't sure whether to put "disk" or "disk storage" into the glossary. Took a stab, but will merit further review. :P